openapi/openapi.yaml

```diff
@@ -1233,6 +1233,303 @@ paths:
           oAuth2ClientCredentials:
             - 'console_rw'
             - 'api_rw'
+  /ad/count:
+    get:
+      tags:
+        - Ads
+      summary: 'Get a count of ads with filter criteria'
+      description: 'Returns a count of ads for the current user and the total size of the result set matching the filter criteria. <br>
+              <br><br>
+              This call supports all the filters in use by GET /ad, except things that do not make sense for a count call, like orderBy, direction, limit and pageToken. 
+              '
+      operationId: getCountOfAdsWithFilters
+      parameters:
+        - name: adIds
+          in: query
+          required: false
+          description: 'List of ad IDs'
+          schema:
+            type: array
+            items:
+              type: integer
+              format: int64
+          example: [ 11232, 2231, 32453 ]
+        - name: titleKeywords
+          in: query
+          required: false
+          description: 'Case-insensitive filter for a keyword in the title; will include only ads which contain the given keyword in their title.'
+          schema:
+            type: string
+            example: 'fiets'
+        - name: status
```

Comment:
as a rule of thumb, I move all objects with a schema description of >2loc, or used in more than one place, to the separate schema files.

Reply:
oh, I copied this from the GET /ad part..